src/dlgRoomSymbol.cpp

```diff
@@ -0,0 +1,220 @@
+                                   .arg(mpSymbols.keys().first());
+        }
+    } else {
+        instructions = tr("Choose:\n"
+                          " • an existing symbol from the list below (sorted by most commonly used first)\n"
+                          " • enter one or more graphemes (\"visible characters\") as a new symbol\n"
+                          " • enter a space to clear any existing symbols\n"
+                          "for all of the %n selected room(s):",
+                          // Intentional comment to separate arguments!
+                          "Use line feeds to format text into a reasonable rectangle if needed, "
+                          "%n is the number of rooms involved.",
+                          mpRooms.size());
+    }
+    label_instructions->setText(instructions);
+}
+
+QStringList dlgRoomSymbol::getComboBoxItems()
+{
+    QHashIterator<QString, unsigned int> itSymbolUsed(mpSymbols);
+    QSet<unsigned int> symbolCountsSet;
+    while (itSymbolUsed.hasNext()) {
+        itSymbolUsed.next();
+        symbolCountsSet.insert(itSymbolUsed.value());
+    }
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
+    QList<unsigned int> symbolCountsList{symbolCountsSet.begin(), symbolCountsSet.end()};
+#else
+    QList<unsigned int> symbolCountsList{symbolCountsSet.toList()};
+#endif
+    if (symbolCountsList.size() > 1) {
+        std::sort(symbolCountsList.begin(), symbolCountsList.end());
+    }
+    QStringList displayStrings;
+    for (int i = symbolCountsList.size() - 1; i >= 0; --i) {
+        itSymbolUsed.toFront();
+        while (itSymbolUsed.hasNext()) {
+            itSymbolUsed.next();
+            if (itSymbolUsed.value() == symbolCountsList.at(i)) {
+                displayStrings.append(tr("%1 {count:%2}",
+                                         // Intentional comment to separate arguments
+                                         "Everything after the first parameter (the '%1') will be removed by processing "
+                                         "it as a QRegularExpression programmatically, ensure the translated text has "
+                                         "` {` immediately after the '%1', and '}' as the very last character, so that the "
+                                         "right portion can be extracted if the user clicks on this item when it is shown "
+                                         "in the QComboBox it is put in.")
+                                              .arg(itSymbolUsed.key())
+                                              .arg(QString::number(itSymbolUsed.value())));
+            }
+        }
+    }
+    return displayStrings;
+}
+
+void dlgRoomSymbol::accept()
+{
+    QDialog::accept();
+    emit signal_save_symbol(getNewSymbol(), selectedColor, mpRooms);
+}
+
+QString dlgRoomSymbol::getNewSymbol()
+{
+    if (mpSymbols.size() <= 1) {
+        return lineEdit_roomSymbol->text();
+    } else {
```

Comment:
:information_source: There is no need for the `else { ... }` as the `return` on the line before will mean that the formerly `else` code can be unindented one level - this is the coding anti-pattern that several *linters* call `else-after-return`... https://clang.llvm.org/extra/clang-tidy/checks/readability-else-after-return.html

Reply:
This one slipped. Corrected.